src/main/kotlin/org/jetbrains/research/testspark/display/utils/ModifiedLinesGetter.kt

```diff
@@ -1,4 +1,4 @@
-package org.jetbrains.research.testspark.display
+package org.jetbrains.research.testspark.display.utils
 
 object ModifiedLinesGetter {
```

Comment:
This file is used in both: `display/` and `action/` folders. Since this class is needed for both phases, it is probably a good idea to take it out of this folder to the top.

Reply:
thank you, fixed here in the #199